src/__tests__/waitFor.test.js

```diff
@@ -55,6 +55,8 @@ test('waits for element until timeout is met', async () => {
   await expect(
     waitFor(() => getByText('Fresh'), { timeout: 100 })
   ).rejects.toThrow();
+
+  await waitFor(() => getByText('Fresh'));
```

Comment:
why adding this?

Reply:
In order to prevent mutation warning after test finishes.

Reply:
```   console.error
      Warning: Can't perform a React state update on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in the componentWillUnmount method.
          in BananaContainer
```

Reply:
Added proper comment